src/components/ec-btn-dropdown/ec-btn-dropdown.vue

```diff
@@ -71,17 +71,17 @@
 </template>
 
 <script setup>
-import { ref } from 'vue';
+import { computed, ref } from 'vue';
 
 import EcBtn from '../ec-btn';
 import EcDropdownSearch from '../ec-dropdown-search';
 
-defineProps({
+const props = defineProps({
   items: {
     type: Array,
     default: () => [],
   },
-  isDisabled: {
+  isBtnDropdownDisabled: {
```

Comment:
Am I right that we could disable button, but have a popover with links available at the same time?